modules/cli/src/main/java/org/apache/ignite/internal/cli/commands/Options.java

```diff
@@ -240,6 +241,15 @@ public static final class Constants {
         /** Unit path option description. */
         public static final String UNIT_PATH_OPTION_DESC = "Path to deployment unit file or directory";
 
+        /** Unit nodes option long name. */
+        public static final String UNIT_NODES_OPTION = "--nodes";
+
+        /** Unit nodes option short name. */
+        public static final String UNIT_NODES_OPTION_SHORT = "-un";
```

Comment:
if I am not mistaken, we decided not to use the two-letter symbols for short options. It might be confusing to understand if it means -u -n or -un.